app/helpers/commodities_helper.rb

```diff
@@ -9,8 +9,11 @@ def leaf_position(commodity)
     end
   end
 
-  def commodity_level(commodity)
-    "level-#{commodity.number_indents}"
+  def commodity_level(commodity, initial_indent)
+    initial_indent ||= 1
```

Comment:
we need `initial_indent ||= 1`, 
since `def commodity_level(commodity, initial_indent=1)`
behaves in a slightly different way: you can explicitly pass ni.